Created header and headline section

peter-akhigbe opened the pull request:
In this branch I did the following:

- Setup HTML/CSS linters
- Created HTML/CSS files
- Created the header and the headline section of the website
- created README.md file in the root of the repository

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,56 @@
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo">My Logo</div>
+        <img class="button" src="image/Enabled.png" alt="" />
+      </nav>
```
- [ ] [REQUIRED] Good job so far! It would be nice if you pay attention to the `top` navigation part. In my opinion, your top `padding` property's value is more than [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=7kEeqOVthntNurgl-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/212690515-1f659192-a496-41d2-82a9-8afaf0549b36.png)

mukhammadrizooff commented on index.html:
```diff
@@ -0,0 +1,56 @@
+        <img class="button" src="image/Enabled.png" alt="" />
+      </nav>
+
+      <div class="content">
```
- [ ] [REQUIRED] Good job so far! The container of the section should be a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. Please change `<div class="content">`(❌) to `<section class="content">` (✔️) 🙏🏼

mukhammadrizooff reviewed (changes requested):
Hi @peter-akhigbe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media3.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

peter-akhigbe commented:
I have updated the following in my code:

- Moving the header closing to line 17
- changed div tag to section tag
- reduced the padding of the nav from 58px to 44px as seen on the Figma design
- applied 'align-items: center' to the nav to make items align properly as seen on the design

Owusu-Desmond commented on index.html:
```diff
@@ -0,0 +1,58 @@
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <section class="section-one">
+      <header>
```
- [ ] Please you should not put the `header` tag in any section so I think it is better if you remove the header from this section since the header is the page header section, not just a section. :pray:

Owusu-Desmond commented on index.html:
```diff
@@ -0,0 +1,58 @@
+    <section class="section-one">
+      <header>
+        <nav>
+          <div class="logo">My Logo</div>
```
- [OPTIONAL]  Please make sure that the logo has a link (a tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. :pray:

Owusu-Desmond reviewed (approved):
Hi @peter-akhigbe , :wave: 

You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.

<img src="https://media1.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif"/>

 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Owusu-Desmond commented:
Please my previous review status changes require but I made a mistake making the status approved :pray:

peter-akhigbe commented:
I have updated the following in my code:

- removed header tag from within section tag
- added link to the logo

huseyinbiyik commented on README.md:
```diff
@@ -1 +1,136 @@
+
+# 📖 Portfolio Project <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
```
[OPTIONAL] Very impressive Readme, indeed 👌 But let's get rid of these kinds of placeholder texts, they are targeting you not the other visitors. 📝

huseyinbiyik commented on index.html:
```diff
@@ -0,0 +1,60 @@
+          </p>
+          <h3>LET'S CONNECT</h3>
+
+          <ul class="social-icons">
+            <li>
+              <a href=""><img src="image/Vector(1).png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="image/Vector(2).png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="image/Vector(3).png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="image/Vector(4).png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="image/Vector(5).png" alt="" /></a>
+            </li>
+          </ul>
```
[OPTIONAL] Your implementation looks great. But let's fix the alignment with Medium icon to make it greater 👨‍🎨

huseyinbiyik reviewed (approved):
Hi @peter-akhigbe,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Alt Text](https://media.giphy.com/media/wZjlCH43M3M0U/giphy.gif)



## Highlights
- Great implementation of the requested changes of previous reviewers 🕸️
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Optional suggestions
- [OPTIONAL] Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
